torch/_inductor/fx_passes/pad_mm.py

```diff
@@ -440,7 +442,11 @@ def realize_symbols(ds):
         ):
             return True
 
-        if not has_triton():
+        scheduling_factory = get_scheduling_for_device(mat1.device.type)
```

Comment:
In addition to the existence of the Triton package, `has_triton()` requires at least one GPU device to be available. However, `get_scheduling_for_device(mat1.device.type)` only requires the existence of the Triton package. It means that `CPU` will run into this pass as well.

Meanwhile, the Triton package will be installed automatically if users install CUDA, Rocm,, or XPU(Intel GPU) torch packages even if a real GPU device does not exist, not mention to the CUDA package is the default wheel package of torch pip channel.

Reply:
Make sense. Thanks